app/components/Icons/Contributor.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Svg, { Path } from 'react-native-svg'
+const Contributor = ({oColor = "#21be93" , dColor = "#212121", style = {height: 24, width: 24}, ...props} = {})=> (
+    <Svg version={1} viewBox="0 0 1138 1138" width={style.width} height={style.height} style={style} {...props}>
+        <Path
+            fill={oColor}
+            d="M450 1047h60a1 1 0 0 0 0-957h-60zm120-807a.9 1 0 0 1 0 660z"
```

Comment:
Can't see the O color when you select the dark theme. Can we change it so that it's black/white depending on the background color?

Reply:
Because I never know when to quit.. I created a shadow component that (on Android) proceeds to do a radial gradient (while repositioning the icon passed in to centre it).

It's kinda bad code, but it does work and the glow effect looks almost identical between iOS and Android.

<img width="1236" alt="screen shot 2018-10-14 at 12 07 48 pm" src="https://user-images.githubusercontent.com/375232/46920973-6f177b80-cfaa-11e8-81de-4d26cfe130d6.png">